src/components/Template/__tests__/Template-test.tsx

```diff
@@ -30,16 +31,16 @@ describe('Template', () => {
       templates: { test: 'it configures compilation <%options%>' },
       data: { options: 'delimiters' },
       useCustomCompileOptions: { test: true },
-      templatesConfig: { compileOptions: { delimiters: '<% %>' } },
+      templatesConfig: { helpers: {}, compileOptions: { delimiters: '<% %>' } },
     });
-    const wrapper = mount(<Template {...props} />);
+    const wrapper = mount((<Template {...props} />) as ReactElementLike);
```

Comment:
why doe it need to be cast? Especially the one from prop-types is confusing to me

Reply:
I took the same approach as in https://github.com/algolia/instantsearch.js/pull/4681.

(Enzyme is waiting for a `ReactElement` but preact returns an `Element`, so we need to cast it)